.travis.yml

```diff
@@ -18,12 +18,17 @@ env:
     - DESTINATION="OS=9.2,name=iPhone 6s"      CONFIGURATION="Debug"
     - DESTINATION="OS=9.2,name=iPhone 6s"      CONFIGURATION="ReleaseTest"
     - DESTINATION="OS=9.3,name=iPhone 6s Plus" CONFIGURATION="Debug"
-    - DESTINATION="OS=9.3,name=iPhone 6s Plus" CONFIGURATION="ReleaseTest"
+    - DESTINATION="OS=9.3,name=iPhone 6s Plus" CONFIGURATION="ReleaseTest"    
+    - DESTINATION="OS=10.0,name=iPhone 6s"      CONFIGURATION="Debug"
+    - DESTINATION="OS=10.0,name=iPhone 6s"      CONFIGURATION="ReleaseTest"
+before_install:
```

Comment:
Travis uses pre-built VM images. The one associated with iOS 10 development is out of date. Not surprising, but the build needs the latest libraries from cocoapods, including all the new release versions of the dependencies, and that image doesn't have them. Telling the VM to update cocoapods let's it find the dependencies.